odoo/http.py

```diff
@@ -668,7 +668,7 @@ def _handle_exception(self, exception):
             error = {
                 'code': 200,
                 'message': "Odoo Server Error",
-                'data': serialize_exception(exception),
+                'data': exception.args and (exception.args[0] or serialize_exception(exception)),
```

Comment:
@duong77476-viindoo như này thôi
```suggestion
                'data': exception.args and (exception.args[0] or ''),
```

Reply:
vâng a